src/main/resources/view/EntryListCard.fxml

```diff
@@ -29,8 +29,8 @@
           </minWidth>
         </Label>
         <Label fx:id="description" styleClass="cell_big_label" text="\$first" wrapText="true" />
-            <Region prefHeight="17.0" prefWidth="17.0" HBox.hgrow="ALWAYS" />
-         <Label fx:id="amount" prefHeight="17.0" prefWidth="56.0" styleClass="cell_big_label" text="\$amount" wrapText="true" />
+            <Region HBox.hgrow="ALWAYS" />
```

Comment:
I temporarily change this to fix the issue of "amount cutting off". Will uncommit this file from the PR so we properly handle this issue in another PR.